### PR TITLE
MODE-2367:Error to access contents of older versions via CMIS

### PR DESCRIPTION
**Pull request by okulikov**
This PR contains:
- Fix for the error accessing previous versions of the document;
- Test case for this scenario

### CODE REVIEW

File: web/modeshape-web-cmis/src/main/java/org/modeshape/cmis/JcrMsRepository.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import org.apache.chemistry.opencmis.jcr.JcrRepository;
+import org.apache.chemistry.opencmis.jcr.JcrTypeManager;
+import org.apache.chemistry.opencmis.jcr.PathManager;
+import org.apache.chemistry.opencmis.jcr.type.JcrTypeHandlerManager;
+
+/**
+ *
+ * @author kulikov
+ */
+public class JcrMsRepository extends JcrRepository {
```
Comment (hchiorean) [inline web/modeshape-web-cmis/src/main/java/org/modeshape/cmis/JcrMsRepository.java]: what's the purpose of this class ? it doesn't seem to be doing anything different than the base class.

**Comment by okulikov** [comment]
@hchiorean , removed that class. I used to debug the problem.

**Comment by hchiorean** [comment]
looks good.
@rhauch  ?

File: web/modeshape-web-cmis/src/main/java/org/modeshape/cmis/JcrMsVersionNode.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        try {
+            return version.getIdentifier();
+        } catch (RepositoryException e) {
+            return null;
+        }
+    }
+
+    @Override
+    protected boolean isLatestVersion() throws RepositoryException {
+        return true;
```
Comment (rhauch) [inline web/modeshape-web-cmis/src/main/java/org/modeshape/cmis/JcrMsVersionNode.java]: Is this correct? Always the latest version?

Comment (okulikov) [inline web/modeshape-web-cmis/src/main/java/org/modeshape/cmis/JcrMsVersionNode.java]: Yes, it is correct. This class belongs to the not versioned nt:version node  type.

**Comment by rhauch** [comment]
Looks good.

**Pull request merged**